docs/UserGuide.md

```diff
@@ -48,9 +48,10 @@ This user guide provides in-depth descriptions of all the available features in
 1. Ensure you have Java `11` or above installed in your Computer.
 
 1. Download the latest `guestbook.jar` from [here](https://github.com/AY2223S1-CS2103T-W16-1/tp/releases).
-
+   ![Ui](images/download.png)
+   ![Ui](images/download2.png)
 1. Copy the file to the folder you want to use as the _home folder_ for your GuestBook.
-
+   ![Ui](images/homefolder.png)
```

Comment:
Should we rename this file too to be more intentional? Such as GuestBookIcon.png